packages/peregrine/src/hooks/__mocks__/useDOMQuery.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+
+import { useDOMQuery } from '../useDOMQuery';
+
+export const TestValidComponent = ({ newText, newHTML, newName }) => {
+    const [, { setInnerText }] = useDOMQuery('#div1');
```

Comment:
Files in the `__mocks__` folder are really to mock functionality. In other words, if `useDOMQuery` did some expensive operation and you wanted to provide a mock implementation that didn't do that expensive operation for testing purposes, you would put that code in this file.
See https://jestjs.io/docs/en/manual-mocks. 

What you have here are more like `fixtures` or mock data, which we usually put into a `__fixtures__` directory instead.

Reply:
Ideally all the logic for the test resides in the actual `.spec.js` test file, not the fixture. The fixture would just return the JSX you're going to test against:

```jsx
    return (
        <React.Fragment>
            <div id="div1">{'I am Test Div 1'}</div>
            <div id="div2" name="oldName">
                {'I am Test Div 2'}
            </div>
        </React.Fragment>
    );
```

If it's small enough (like this one), you can inline it directly into the `.spec.js` file as well.